pom.xml

```diff
@@ -237,6 +216,8 @@
       <id>spark-ut</id>
       <modules>
         <module>gluten-ut</module>
+        <module>gluten-ut/common</module>
+        <module>gluten-ut/${spark.shim.version}</module>
```

Comment:
Thanks for your improvement!
Although same module names (spark32/spark33/spar34) are used, it may be better to define & use `spark.test.module` instead of `spark.shim.version` for readability since they are under gluten-ut/, not shims. Right?

Reply:
added spark.test.module to decouple with spark.shim.module